dockerfileTemplates/Dockerfile.nodejs12

```diff
@@ -3,7 +3,7 @@ FROM node:12
 COPY package*.json ./
 
 RUN npm set progress=false
-RUN {{buildCommand}}
+RUN --restart unless-stopped {{buildCommand}}
```

Comment:
Ahh, I don't think you can put that in the RUN. Will probably need to go on the common/performDeployment

Reply:
ah ok